src/components/ui/CaptionInput.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+
+import CircleIcon from './CircleIcon'
+import IconButton from './IconButton'
+import Input from './Input'
+import ListEntry from './ListEntry'
+
+const StyledCaptionInput = styled(ListEntry)`
+  padding: 8px;
+  height: 33px;
+  display: flex;
+  justify-content: space-between;
```

Comment:
Just curious, why is the display: flex, space-between necessary?

Reply:
I originally put it in there because I was playing around with the spacing, but I don't actually need it.